pkg/aqua/command.go

```diff
@@ -46,6 +61,11 @@ func (c *command) Exec(imageRef string) (report ScanReport, err error) {
 		}
 	}()
 
+	image := imageRef.WithDigest()
+	if c.cfg.UseImageTag {
+		image = imageRef.WithTag()
```

Comment:
Isn't it necessary to test `WithTag`? In tests, `UseImageTag` always appears to be false.

Reply:
This will be done by a dedicated PR.